packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -611,12 +611,10 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
   // Only call this from places that will definitely trigger a rebuild.
   void _updatePosition() {
     _configuration = widget.scrollBehavior ?? ScrollConfiguration.of(context);
-    _physics = _configuration.getScrollPhysics(context);
-    if (widget.physics != null) {
-      _physics = widget.physics!.applyTo(_physics);
-    } else if (widget.scrollBehavior != null) {
-      _physics = widget.scrollBehavior!.getScrollPhysics(context).applyTo(_physics);
-    }
+    final ScrollPhysics? physicsFromWidget = widget.physics ?? widget.scrollBehavior?.getScrollPhysics(context);
+    _physics =  _configuration.getScrollPhysics(context);
+    _physics = physicsFromWidget?.applyTo(_physics) ?? _physics;
```

Comment:
```suggestion
    _physics = physicsFromWidget?.applyTo(_physics) ?? _configuration.getScrollPhysics(context);
```

Reply:
hmm if I apply the suggested change, is `_physics` going to be up to date for the expression before `??`

Reply:
oh I didn't realize the applyTo uses _physics, in that case no, the suggestion is wrong, the original call is fine